.circleci/config.yml

```diff
@@ -23,26 +23,22 @@ jobs:
       - setup_ci_image
       - run: inv test
       - run: cp manifests/metallb.yaml manifests/metallb.yaml.prev
-  e2etest-bgp:
+  e2etest:
+    parameters:
+      ip-family:
+        type: string
+      protocol:
+        type: string
     machine:
       image: ubuntu-2004:202104-01
     steps:
       - setup_ci_image
       - setup_kind
       - run: inv build
-      - run: inv dev-env -p bgp
-      - run: inv e2etest -e /tmp/kind_logs
-      - store_artifacts:
-          path: /tmp/kind_logs
-  e2etest-layer2:
-    machine:
-      image: ubuntu-2004:202104-01
-    steps:
-      - setup_ci_image
-      - setup_kind
-      - run: inv build
-      - run: inv dev-env -p layer2
-      - run: inv e2etest -e /tmp/kind_logs
+      - run: inv dev-env -p << parameters.protocol >> -i  << parameters.ip-family >>
+      - run: |
+          if [ "<< parameters.protocol >>" = "layer2" ]; then FOCUS=L2; else FOCUS=BGP; fi
```

Comment:
to avoid this `if` could we just match the `FOCUS` names to the values in the matrix below?

Reply:
That was the original implementation.
The way exclude works however, is that it requires an item for each permutation to exclude.
I can't say 
```
            exclude:
              - protocol: bgp
                ip-family: ipv6
```
I would also need to expand with different FOCUS-es which was as much as ugly. Also, those two parameters would always be coupled, so I think it's better to have only one saying which mode we are deploying / testing.

On top of that, this part is gonna disappear when https://github.com/metallb/metallb/pull/931 lands